src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
   [here](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2979#issuecomment-1061060541).
   ([#2982](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2982))
 
+* Bumped minimum required gRPC version.
```

Comment:
nice to include the new and old minimum version right in the changelog, to make it easier to know that, w/o having to open the PR.

Reply:
Updated with GrpcPkgVer values.